### ISSUE TITLE
Misplaced repair interaction points for helicopter rotors

### ISSUE THREAD

**Issue by Ivanowicz** [acemod/ACE3#9558]
**Mods:**
- **Arma 3:** `2.14`
- **CBA:** `3.16.0`
- **ACE3:** `3.16.0`
- **RHS** `0.5.6`

**Description:**
Interaction points for most RHS helicopters Main Rotors are located at the tip of one of the blades, or in the middle of one, making them rather difficult to find, especially for big helicopters. I could not find the Main Rotor point on CH-53E, but I could have simply missed it due to size of the vehicle. Except for the RAH-66 and Mi-290, which have the repair points at the tip of one of the blades, all vanilla helicopters have rotor repair points in proper locations. 

Repair points for Tail Rotors of RHS helis seem to be missing completely, except for the AH-6M and UH-1H. RHS Ka-52 and vanilla Mi-290 have the "Tail Rotor" (upper rotor) repair point at the end of a blade.

This also made me notice that when looking for repair points, you need to be looking towards the center of the vehicle, which can be slightly confusing when dealing with large and long vehicles like helis - should I put that in a separate issue?

**Expected behavior:**
Repair points for rotors appearing either at the center of the rotor, or close enough that they're easy to find.

**Where did the issue occur?**
- Dedicated / Singleplayer

**Log Files:**
- https://gist.github.com/Ivanowicz/00deb748b1104fa6a1b97e0e15ef23f2

**Screenshots:**
![AH-1Z](https://github.com/acemod/ACE3/assets/28870740/5a75e5e4-257f-4c3e-a2c3-edcc6b9666f4)
![RAH-66](https://github.com/acemod/ACE3/assets/28870740/140a72e9-efdd-40c5-8ab6-b20be3ea2b02)

**Comment by jonpas** [comment]
We just use the selections for those, that's just how they are placed in the model. I don't think manually specifying those makes sense, that blade is part of the rotor anyways.

**Comment by rautamiekka** [comment]
> that blade is part of the rotor anyways.

Yeah, you can't damage single blades, it's the whole rotor or nothing.

**Comment by PabstMirror** [comment]
maybe look into using `AveragePoint` for all hitpoints?
but this might be worse for things like hithull??

**Comment by Ivanowicz** [comment]
> maybe look into using `AveragePoint` for all hitpoints? but this might be worse for things like hithull??

Just checked and this seems to be the solution - it fixes all the problems with Main Rotor locations (_at worst_ the repair point is about .5 m to the side). It does seem to mess up repair points of sights, lights and mg mounts on some tanks, so it probably should be limited to rotors only. I'll try to put up a pull request sometime tomorrow.

I also finally figured out why the Tail Rotor point was not appearing on most RHS helis - it has the parameter `depends` set to tail, so the rotor gets ignored. Repairing the tail fixes the rotor, meaning this is only a UI issue. I think the best solution would be to modify the name of the tail repair point to something like _Tail/Tail Rotor_ if it has a dependant rotor.
